### ISSUE TITLE
Platypus: Page content not contained by landmarks

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#469]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, not all page content is contained by landmarks.

---

https://dequeuniversity.com/rules/axe/4.3/region?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A

**Comment by micpap25** [comment]
Hi, I cannot find the referenced page content that is not contained by landmarks; has this issue been resolved?

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
